platforms/web/src/containers/AppRoutes/AppRoutes.tsx

```diff
@@ -53,7 +53,7 @@ export default function AppRoutes() {
   useNotifications();
 
   if (userData.user && !userData.loading && window.location.href.includes('#token')) {
-    return <Navigate to="/" />; // component instead of hook to prevent extra re-renders
+    return <Navigate to={`/${location.search}`} />; // component instead of hook to prevent extra re-renders
```

Comment:
Do we also need this change?

Reply:
Yes, otherwise being redirected to `u=personal-details#token=AAABBBCCC` wouldn't make a difference if we keep this navigating `to="/"`. Our goal is to open the `personal-details` modal when we're redirected from social media login.

Reply:
Maybe we can use `PATH_HOME` here:
```javascript
createURL(`${PATH_HOME}${location.search}`, {});
```

Reply:
I'm reading this just now and think if the only reason is to remove the token, we don't need a navigate at all. I also feel this breaks the expectations when a user logs in on a media page, for example.

`/m/1234/movie-i-want-to-watch` -> `social login` -> `back on home page :-(`

Reply:
@mirovladimitrovski can you test the flow above

Reply:
@dbudzins  @ChristiaanScheermeijer I didn't really question the navigate at all because it was already there and I just added the search params on top of it, but you're right about what you're describing and we need to create a ticket for this.

Reply:
@dbudzins @ChristiaanScheermeijer Here it is:
https://github.com/jwplayer/ott-web-app/pull/499